docs/officers/payments.rst

```diff
@@ -0,0 +1,9 @@
+Payments
+========
+
+Apiary supports several payment methods for both :doc:`dues/index` and :doc:`travel`.
+
+.. todo::
```

Comment:
TODO docs without a corresponding issue or something is kind of pointless, and frustrating as a user because when you look at the sidebar, it looks there will be real docs on this page

Reply:
I removed references to the stub documents.